### ISSUE TITLE
Slow if work with custom inspector

### ISSUE THREAD

**Issue by roygear** [mackysoft/Unity-SerializeReferenceExtensions#6]
There is a performance issue if that type has a custom inspector and a field opened.

```C#
[CustomEditor(typeof(Example))]
public class ExampleEditor : UnityEditor.Editor {
    //issue happen even no extra content
    public override void OnInspectorGUI()
    {
        base.OnInspectorGUI();
    }
}
```
![Untitled-1](https://user-images.githubusercontent.com/30362268/156794461-6228474c-9b43-4a90-821c-3b887c9520e3.jpg)

Tested by Unity 2020.3.30

**Comment by mackysoft** [comment]
Thanks for the feedback!
I was able to reproduce it. Will investigate and fix.

![image](https://user-images.githubusercontent.com/13536348/156796543-9204c0a1-6a7e-4c73-964f-0b9878cd7b2f.png)

**Comment by roygear** [comment]
thanks

**Comment by mackysoft** [comment]
I have reported it to Issue Tracker as it may be a bug in Unity Editor.
https://fogbugz.unity3d.com/default.asp?1408562_uivd8c902qn3ooa3

**Comment by mackysoft** [comment]
I have released an update with the optimizations I can make at this time.
https://github.com/mackysoft/Unity-SerializeReferenceExtensions/releases

The issue reported to IssueTracker is still waiting.

**Comment by mackysoft** [comment]
Unity QA responded back.

> Thanks for reporting this issue, after some investigation we have found that it's already fixed in Unity 2020.3.31f1 and above.

This issue appears to be fixed in Fix Unity 2020.3.31f1!
